src/org/zaproxy/zap/extension/openapi/ExtensionOpenApi.java

```diff
@@ -240,5 +249,59 @@ public URL getURL() {
             return null;
         }
     }
+    
+    private CommandLineArgument[] getCommandLineArguments() {
+        arguments[ARG_IMPORT_FILE_IDX] = new CommandLineArgument("-openapifile", 1, null, "", 
+                "-openapifile <path>      " + Constant.messages.getString("openapi.cmdline.file.help"));
+        arguments[ARG_IMPORT_URL_IDX] = new CommandLineArgument("-openapiurl", 1, null, "", 
+                "-openapiurl <url>        " + Constant.messages.getString("openapi.cmdline.url.help"));
+        return arguments;
+    }
+
+    @Override
+    public void execute(CommandLineArgument[] args) {
+        if (arguments[ARG_IMPORT_FILE_IDX].isEnabled()) {
+            for (CommandLineArgument arg : args) {
+                Vector<String> params = arg.getArguments();
+                if (params != null) {
+                    for (String file : params) {
```

Comment:
This for loop should be `for (String file : args[ARG_IMPORT_FILE_IDX].getArguments()) {` instead, to just use the arguments of import file.